### PR TITLE
Review and update the testing doc page

### PR DESCRIPTION
**Pull request by praneesha**
Review and update the testing doc page.

### CODE REVIEW

File: docs/TESTS.md [hunk 1]
```diff
@@ -1,43 +1,70 @@
...
+```
 
-     ```sh
-     bal test --offline
-     ```
+### Test exercise with multiple tests
+
+Follow the steps below to test an exercise with multiple tests.
+
+1. Once the first test passes, enable the next test by commenting the `enable: false` property from the `@test:Config` annotation below of it as shown in the example below.
```
Comment (ThisaruGuruge) [inline docs/TESTS.md:45]: Is commenting out the line the correct way? Shouldn't we ask them to remove the line completely?

Comment (praneesha) [reply docs/TESTS.md]: Changed to removing.

File: docs/TESTS.md [hunk 2]
```diff
@@ -1,43 +1,70 @@
...
      ```sh
-     bal init
+     cd ~/exercism/ballerina/hello-world/
      ```
+
+2. Follow the provided instructions and develop a solution for the exercise.
+
+## Test your solution
+
+Some Ballerina exercises will have a single test while others will have multiple tests. The tests can be found in the respective test file of the `/tests` directory inside the root directory of the exercise.
```
Comment (anupama-pathirage) [inline docs/TESTS.md:29]:
```suggestion
Some Ballerina exercises will have a single test, while others will have multiple tests. The tests can be found in the respective test file of the `/tests` directory inside the root directory of the exercise.
```

File: docs/TESTS.md [hunk 3]
```diff
@@ -1,43 +1,70 @@
...
-     ```
+### Test exercise with multiple tests
+
+Follow the steps below to test an exercise with multiple tests.
+
+1. Once the first test passes, enable the next test by removing the `enable: false` property shown in the example below from the `@test:Config` annotation of it.
+
+     ```ballerina
+     @test:Config {
+     enable: false
```
Comment (malinthar) [inline docs/TESTS.md:49]:
```suggestion
       enable: false
```

File: docs/TESTS.md [hunk 4]
```diff
@@ -1,43 +1,70 @@
...
-2. Change directory to the newly downloaded exercise
+Open a Command Prompt and execute the command below to download the exercise.
 
-     ```sh
-     cd ~/exercism/ballerina
-     ```
+>**Tip:** Replace the value of the `--exercise` property with the name of your preferred exercise.
+
+```sh
+exercism download --track ballerina --exercise hello-world
```
Comment (anupama-pathirage) [inline docs/TESTS.md:12]:
```suggestion
exercism download --track ballerina --exercise <exercise name>
```

File: docs/TESTS.md [hunk 5]
```diff
@@ -1,43 +1,70 @@
...
-1. Open a Command Prompt and get the first exercise:
+Follow the steps below to develop a solution for an exercise and test it locally.
 
-     ```sh
-     exercism download --track ballerina --exercise hello-world
-     ```
+## Download the exercise
 
-2. Change directory to the newly downloaded exercise
+Open a Command Prompt and execute the command below to download the exercise.
```
Comment (anupama-pathirage) [inline docs/TESTS.md:7]:
```suggestion
Open a Command Prompt and execute the command below with the name of the preferred exercise to download the exercise.
```

File: docs/TESTS.md [hunk 6]
```diff
@@ -1,43 +1,70 @@
...
+```
 
-     ```sh
-     bal test --offline
-     ```
+### Test exercise with multiple tests
+
+Follow the steps below to test an exercise with multiple tests.
+
+1. Once the first test passes, enable the next test by removing the `enable: false` property shown in the example below from the `@test:Config` annotation of it.
```
Comment (anupama-pathirage) [inline docs/TESTS.md:45]: `shown in the example below from the `@test:Config` annotation of it.` bit too wordy

File: docs/TESTS.md [hunk 7]
```diff
@@ -1,43 +1,70 @@
...
-    By default, only the first test is enabled.
-    This is intentional as it allows you to focus on just making that one test pass.
-    Once it passes, you can enable the next test by removing `enable: false` from the test's `@test:Config {}` set.
-    When all tests have been enabled ans your implementation makes them all pass, you'll have solved the exercise!
+   ```sh
+     bal test
+   ```
 
-Good luck!  Have fun!
+3. Repeat the two steps above to enable the remaining tests one by one and run them.
```
Comment (anupama-pathirage) [inline docs/TESTS.md:61]: Why do we need to ask the users to run tests one by one? IMO it is unnecessary.

File: docs/TESTS.md [hunk 8]
```diff
@@ -1,43 +1,70 @@
...
 
-If you get stuck, at any point, don't forget to reach out for help:
-- on [the forum][forum],
-- in the #support channel on [Exercism's discord server][discord].
+You have solved the exercise when all tests have been enabled and your implementation makes them all pass.
```
Comment (anupama-pathirage) [inline docs/TESTS.md:63]: You have successfully completed the exercise by enabling all tests and ensuring that your implementation passes each one.
```suggestion
You have successfully completed the exercise by enabling all tests and making sure that your implementation passes each one.
```

File: docs/TESTS.md (hunk 7 above)
Comment (praneesha) [reply docs/TESTS.md:61]: That is the approach we have followed when designing Exercism exercises. @malinthar - Can you verify?

File: docs/TESTS.md [hunk 9]
```diff
@@ -1,43 +1,70 @@
...
-     ```
+## Download the exercise
 
-2. Change directory to the newly downloaded exercise
+Open a Command Prompt and execute the command below to download the exercise.
 
-     ```sh
-     cd ~/exercism/ballerina
-     ```
+>**Tip:** Replace the value of the `--exercise` property with the name of your preferred exercise.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply docs/TESTS.md:9]: Removed.

File: docs/TESTS.md (hunk 6 above)
Comment (praneesha) [reply docs/TESTS.md:45]: Changed to `Once the first test passes, enable the next test by removing the `enable: false` property from the `@test:Config` annotation of it.`

File: docs/TESTS.md (hunk 7 above)
Comment (malinthar) [reply docs/TESTS.md:59]: Exercism encourages test driven development. That means enable one test case and get it passed by gradually improving your solution(Not the case for every exercise). We don't have to specifically say to enable the tests one by one. We can just say `Repeat the two steps above to enable the remaining tests.`

File: docs/TESTS.md (hunk 6 above)
Comment (anupama-pathirage) [reply docs/TESTS.md]: IMO, a simple sentence as follows is good enough. 
```Once the first test passes, enable the next test by removing the `enable: false` property of the `@test:Config` annotation. ```

Comment (praneesha) [reply docs/TESTS.md]: Updated.

File: docs/TESTS.md (hunk 7 above)
Comment (praneesha) [reply docs/TESTS.md:59]: Updated.

**Review by malinthar — APPROVED** [review]

**Review by anupama-pathirage — APPROVED** [review]

**Review by ThisaruGuruge — APPROVED** [review]
